docs/components/power-sensor/_index.md

```diff
@@ -0,0 +1,239 @@
+
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **Code sample** tab, select your preferred programming language, and copy the sample code generated.
+
+{{% snippet "show-secret.md" %}}
+
+When executed, this sample code will create a connection to your robot as a client.
+Then control your robot programmatically by adding API method calls as shown in the following examples.
+
+These examples assume you have a power sensor called `"my_power_sensor"` configured as a component of your robot.
+If your power sensor has a different name, change the `name` in the code.
+
+Be sure to import the power sensor package for the SDK you are using:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python
+from viam.components.powersensor import powersensor
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go
+import (
+  "go.viam.com/rdk/components/powersensor"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## API
+
+The power sensor component supports the following methods:
+
+{{< readfile "/static/include/components/apis/power-sensor.md" >}}
+
+### GetCurrent
+
+Return the current of a specified device and whether it is AC or DC.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional[Dict[str,Any]])](https://docs.python.org/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+- `timeout` [(Optional[float])](https://docs.python.org/3/library/typing.html#typing.Optional): Specify a time limit (in seconds) for how long `get_current` should wait for a response.
+
+**Returns:**
+
+- [(Tuple[float, bool])](https://docs.python.org/3/library/functions.html#float): A float representing the current reading in amps. A bool indicating whether the current is AC (True) or DC (False).
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/power_sensor/power_sensor/index.html#viam.components.power_sensor.power_sensor.PowerSensor.get_current).
+
+```python
+# Get the current reading from the power sensor
+current, is_ac = await my_power_sensor.get_current()
+print("The current is ", current, " A, Is AC: ", is_ac)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [Context](https://pkg.go.dev/context): Control the lifecycle of the operation by handling timeouts and managing cancellations.
+- `extra`[(Optional[Dict[str, Any]])](https://docs.python.org/3/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+
+**Returns:**
+
+- `float64`: The measurement of the current, represented as a 64-bit float number.
+- `bool`: Indicate whether current is AC (True) or DC (False).
+- `error`: Report any errors that might occur during operation. Successful operation will be `nil`.
```

Comment:
nit: we use a convention of a single line per sentence. Also, I've proposed a slight rephrase of the second sentence if you agree:

```suggestion
- `error`: Report any errors that might occur during operation.
   For a successful operation, `error` returns `nil`.
```